src/operation/in_memory_model_store.jl

```diff
@@ -112,22 +97,26 @@ Base.flush(store::InMemoryModelStore) = nothing
 #     return func(store)
 # end
 
-function initialize_model_storage!(
+function initialize_storage!(
     store::InMemoryModelStore,
     container::OptimizationContainer,
     params::ModelStoreParams,
 )
-    @debug "initialize_model_storage"
+    @debug "initialize_storage"
 
     num_of_executions = get_num_executions(params)
     for type in STORE_CONTAINERS
         field_containers = getfield(container, type)
         store_container = getfield(store.data, type)
-        for (key, container) in field_containers
-            container_axes = axes(container)
+        for (key, field_container) in field_containers
+            container_axes = axes(field_container)
             @debug "Adding $(encode_key_as_string(key)) to InMemoryModelStore"
             if length(container_axes) == 2
-                column_names = string.(axes(container)[1])
+                if type == STORE_CONTAINER_PARAMETERS
+                    column_names = string.(get_parameter_array(field_container).axes[1])
```

Comment:
Let me know if there is a better way to handle this.

Reply:
I don't have a better idea at the moment except enforcing that the axes are always strings.